### PR TITLE
Export button, continued

### PR DESCRIPTION
**Pull request by jklynch**
continuation of #35

### CODE REVIEW

**Comment by jklynch** [comment]
I have been trying to use traitlets.config features to demonstrate adding a suitcase to bluesky-browser through the bluesky_browser_config.py file. I added these lines to bluesky_browser_config.py:
```
c = get_config()
c.export_widget.suitcases = ['csv', 'xdi']
```
The result is an error in traitlets.config.Application.application.load_config_file(), before any actual configuration is attempted:
```
...
TypeError: 'LazyConfigValue' object is not iterable
```
The cause seems to be capitalization. There is no error if the config file is changed to this:
```
c = get_config()
c.Export_widget.suitcases = ['csv', 'xdi']
```

**Comment by danielballan** [comment]
Odd! In any case I think all the class names should be made PEP8-compliant, e.g. `class export_widget` -> `class ExportWidget`.

File: bluesky_browser/export.py [hunk 1]
```diff
@@ -0,0 +1,477 @@
...
+                self.main_layout.addRow(trait_type.metadata["display_label"], widget)
+            else:
+                # no widget to create for this trait
+                pass
+
+        # now create widgets for the suitcase-specific traits
+        suitcase_specific_traits = self.model.get_ordered_traits()
+        print(suitcase_specific_traits)
+
+        for trait_name, trait_type in suitcase_specific_traits.items():
```
Comment (danielballan) [inline bluesky_browser/export.py:312]: This loop and the one above can be one loop over `self.model.traits()`.

Comment (danielballan) [reply bluesky_browser/export.py:312]: ...or two loops over `self.model.traits()` and `self.class_own_traits()`.

Comment (danielballan) [reply bluesky_browser/export.py:312]: `trait_names()` and `class_trait_names()` might be useful here too.

**Comment by danielballan** [comment]
We're going to revisit the proposal of a global dict mapping trait types to QWidget subclasses. These subclasses should implement standardized methods for `set_default` and `subscribe`, as we are starting to do in mily anyway.

Thus, a user writes a class with traitlets, and a QWidget can automatically be assigned. The user can override/supplement our global dict by assigning a `tag` to the `traitlet`. No need to modify the dict.

**Comment by danielballan** [comment]
To be continued elsewhere -- maybe in Xi-cam.gui?

**Pull request closed without merging**